EIPS/eip-1.md

```diff
@@ -42,19 +42,23 @@ The EIP process begins with a new idea for Ethereum. It is highly recommended th
 
-If the EIP collaborators approve, the EIP editor will assign the EIP a number (generally the issue or PR number related to the EIP), label it as Standards Track, Informational, or Meta, give it status “Draft”, and add it to the git repository. The EIP editor will not unreasonably deny an EIP. Reasons for denying EIP status include duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the Ethereum philosophy.
+Drafts must be formatted as described in this EIP; a template is available in [eip-X.md](https://github.com/ethereum/EIPs/blob/master/eip-X.md). All required headers must be present and filled in correctly. For a draft, the `eip` field must match the filename, without the 'eip-' prefix or '.md' suffix (eg, for `eip-draft-vitalik-foo.md`, the `eip` field should read `draft-vitalik-foo`). Drafts must contain a `discussions-to` header, specifying the URL to a venue where collaborators can discuss the draft. This may be a Github issue in the EIPs repository, or any other venue as the author sees fit.
 
-Standards Track EIPs consist of three parts, a design document, implementation, and finally if warranted an update to the [formal specification]. The EIP should be reviewed and accepted before an implementation is begun, unless an implementation will aid people in studying the EIP. Standards Track EIPs must be implemented in at least three viable Ethereum clients before it can be considered Final.
+An automated process will check all pull requests; if the request passes the format checks and contains only additions or edits to drafts owned by the submitter, it will be automatically merged. No editorial control is exercised over the content of an EIP draft.
 
-For an EIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+Standards track EIPs of type 'Core' are updated from 'Draft' to 'Accepted' status when a core devs meeting has accepted the draft and the participants have announced their intention to implement the EIP for inclusion in a future hard fork. At that point, an EIP number is assigned by the editors and the EIP is moved to the EIPS directory and renamed accordingly. Once a Core EIP has been implemented in at least three clients, and all pass a common set of test suites, the status of the EIP is updated to 'Final'. An EIP can also be assigned status “Deferred”. The EIP author or editor can assign the EIP this status when no progress is being made on the EIP.
```

Comment:
I'm assuming that only EIPs that are in a Final state are actually included in a hard fork, although this isn't explicitly stated here.

Is there a process where Accepted EIPs that are planned to be included in a hard fork are removed from inclusion for any reason? It might be helpful to elaborate on that.

Reply:
Whether an EIP ends up in a particular hard fork is down to whether it's in the "Hard fork meta" EIP for that HF. Suggestions on wording for documenting that are welcome.